gulpfile.js

```diff
@@ -61,13 +61,33 @@ gulp.task('tslint', function () {
         .pipe(tslint.report('verbose'));
 });
 
+function readArgument(argumentName) {
+    var argumentNameIndex = process.argv.indexOf("--" + argumentName);
+    var argumentValueIndex = argumentNameIndex + 1;
+    return argumentNameIndex > -1 && argumentValueIndex < process.argv.length
+        ? process.argv[argumentValueIndex]
+        : null;
+}
+
 function test() {
+    // Defaults
+    var invert = true;
+
+    // Check if arguments were passed
+    var pattern = readArgument("pattern");
+    if (pattern !== null) {
+        invert = false;
+        console.log("\nTesting cases that match pattern: " + pattern);
```

Comment:
Would it be good to see this even for the default pattern? Just so that it's clearer if we're quickly reading the log